server/src/main/java/org/elasticsearch/search/aggregations/metrics/GeoBoundsAggregator.java

```diff
@@ -36,20 +36,20 @@
 import java.io.IOException;
 import java.util.Map;
 
-final class GeoBoundsAggregator extends MetricsAggregator {
+public class GeoBoundsAggregator extends MetricsAggregator {
```

Comment:
Does it need to be public?

Reply:
no. this was left from when I tried to inherit from GeoBoundsAggregator.

I've reverted this file to how it was in master, and fixed some other `protected` instance variables to be private. good catch. thanks!